build.gradle

```diff
@@ -1,16 +1,43 @@
 plugins {
-    id 'io.spring.dependency-management' version '1.0.3.RELEASE'
-    id 'org.springframework.boot' version '1.5.6.RELEASE'
+  id 'checkstyle'
+  id 'findbugs'
+  id 'org.sonarqube' version '2.5'
+  id 'jacoco'
+  id 'io.spring.dependency-management' version '1.0.3.RELEASE'
+  id 'org.springframework.boot' version '1.5.6.RELEASE'
+  id 'org.owasp.dependencycheck' version '1.4.5.1'
+  id 'com.github.ben-manes.versions' version '0.15.0'
```

Comment:
Perhaps update README.md on how to use those

Reply:
done